tests/unit/test.program.js

```diff
@@ -417,6 +417,17 @@ describe('program.main', () => {
         assert.equal(customPrefs.prop2, 'value2');
       });
   });
+
+  it('pass shouldExitProgram option to commands', () => {
+    const fakeCommands = fake(commands, {
+      lint: () => Promise.resolve(),
+    });
+    return execProgram(['lint'], {commands: fakeCommands}
+    ).then(() => {
+      const options = fakeCommands.lint.firstCall.args[1];
+      assert.equal(options.shouldExitProgram, false);
```

Comment:
Same thing here: this needs to be `assert.strictEqual` otherwise many things will match `false`.

Thanks for adding this test!